core/src/test/java/com/yahoo/oak/ComputeTest.java

```diff
@@ -29,12 +34,56 @@
     private static final int KEY_SIZE = 10;
     private static final int VAL_SIZE = Math.round(5 * K);
 
-    private static OakMap<ByteBuffer, ByteBuffer> oak;
+    private static ConcurrentZCMap<ByteBuffer, ByteBuffer> oak;
 
     private static int numOfEntries;
 
     ExecutorUtils<Void> executor;
     private CountDownLatch latch;
+    private Supplier<ConcurrentZCMap> builder;
+
+    public ComputeTest(Supplier<ConcurrentZCMap> supplier) {
+        this.builder = supplier;
+    }
+
+    @Parameterized.Parameters
+    public static Collection parameters() {
+
+        Supplier<ConcurrentZCMap> s1 = () -> {
+            ByteBuffer minKey = ByteBuffer.allocate(KEY_SIZE * Integer.BYTES);
```

Comment:
This is because for now the setup of builder of OakHashMap and OakMap is same  this is might not be like that when the full implmention will be exsits